src/Components/Footer.jsx

```diff
@@ -1,18 +1,59 @@
+import React from "react";
+import { Grid, Box, Container } from "@material-ui/core";
+import { List, ListItem } from "@material-ui/core";
+import { Stack, Typography } from "@mui/material";
+import { Divider } from "@mui/material";
+
+import democracylabLogo from "../Assets/democracylab-logo.png";
+import openseattleLogo from "../Assets/openseattle-logo.png";
+
 const Footer = () => {
   return (
-    <footer className="footer-cont">
-      <div className="info-cont">
-        <a className="white-link" href="mailto:cvp@seamus.work">
-          <ul>Email Us</ul>
-        </a>
-        <a className="white-link" href="https://www.democracylab.org/about">
-          <ul>Democracy Lab</ul>
-        </a>
-        <a className="white-link" href="https://openseattle.org/about/">
-          <ul>Open Seattle</ul>
-        </a>
-      </div>
-    </footer>
+    <Container maxWidth="xl">
+      <Stack direction="row" sx={{ display: "flex", padding: 5 }}>
+        <Grid container columns={2} spacing={6} justifyContent="space-between">
+          <Grid item>
+            <List>
+              <ListItem>
+                <a href="/" style={{ color: "#000000" }}>
+                  Home
+                </a>
+              </ListItem>
+              <ListItem>
+                <a href="/calculator" style={{ color: "#000000" }}>
+                  Calculator
+                </a>
+              </ListItem>
+              <ListItem>
+                <a href="/contact" style={{ color: "#000000" }}>
+                  Contact
+                </a>
+              </ListItem>
+              <ListItem>
+                <a href="/about" style={{ color: "#000000" }}>
+                  About
+                </a>
+              </ListItem>
+            </List>
+          </Grid>
+
+          <Grid item sx={{ float: "right", position: "relative" }}>
+            <Stack direction="column">
+              <img src={openseattleLogo} alt={"Open Seattle Logo"} />
+              <img src={democracylabLogo} alt={"Democracy Lab Logo"} />
+            </Stack>
+          </Grid>
+        </Grid>
+      </Stack>
+      <Divider sx={{ marginTop: 1, marginBottom: 1 }} />
+      <Box justifyContent="center" marginBottom={1}>
+        <Typography align="center" fontFamily={"sans-serif"} fontSize="14px">
+          {" "}
+          The information on this site is not, nor should it be, considered
```

Comment:
How do you feel about using the custom `Text` component that is in the `ui-kit`? That way we can pass any styles, font faces, text content as props.

Reply:
I can try but i was getting weirdness with the v5 + v4 of mui interacting. Like the accordion situation.